configuration/front_controllers_and_kernel.rst

```diff
@@ -85,7 +85,7 @@ you must implement them all:
 :method:`Symfony\\Component\\HttpKernel\\KernelInterface::registerBundles`
     It must return an array of all bundles needed to run the application.
 
-:method:`Symfony\\Bundle\\FrameworkBundle\\Kernel\\MicroKernelTrait::configureRoutes`
+:method:`Symfony\\Bundle\\FrameworkBundle\\Kernel\\MicroKernelTrait::configureRouting`
     It adds individual routes or collections of routes to the application (for
     example loading the routes defined in some config file).
```

Comment:
I think we need a `.. versionadded:: 5.1` here to indicate that this method was introduced in 5.1

Reply:
Done! Did I get you correctly?

Reply:
Yep perfect!